src/test/java/net/datafaker/providers/base/AddressTest.java

```diff
@@ -261,4 +261,10 @@ void nonDefaultLocaleStreetName(String locale) {
         BaseFaker localFaker = new BaseFaker(new Locale(locale));
         assertThat(localFaker.address().streetName()).isNotEmpty();
     }
+
+    @RepeatedTest(100)
+    void dutchAddress() {
+        assertThat(NL_FAKER.address().stateAbbr()).matches("[A-Z]{2}");
+        assertThat(NL_FAKER.address().fullAddress()).matches("[A-Z].+, [0-9]{4} [A-Z]{2}, [A-Z].+");
```

Comment:
should it check for only uppercase?

Reply:
it should check that the fullAddress starts with an uppercase, followed by anything.

Reply:
(I've ran this test approx 1_000_000 times, but I thought that was a bit too much to commit)

Reply:
ok, i see, thanks